docs/iris/src/userguide/real_and_lazy_data.rst

```diff
@@ -0,0 +1,225 @@
+.. _real_and_lazy_data:
+
+==================
+Real and Lazy Data
+==================
+
+What is Real and Lazy Data?
+---------------------------
+
+Every Iris cube contains an n-dimensional  data array, which could be real or
+lazy.
+
+Real data is contained in an array which has a shape, a data type, some other
+useful information and many data points, each of which use up a small
+allocation of memory.  This generally takes the form of a numpy array.
+
+Lazy data is contained in a conceptual array which retains the information
+about its real counterpart but has no actual data points, so its memory
+allocation is much smaller.  This will be in the form of a Dask array.
+
+Arrays in Iris can be converted flexibly(?) between their real and lazy states,
+although there are some limits to this process.  The advantage of using lazy
+data is that it has a small memory footprint, so certain operations
+(such as...?) can be much faster.  However, in order to perform other
+operations (such as calculations on actual data values) the real data must be
+realized.
+
+* When a cube is loaded, are the data arrays always lazy to begin with? *
```

Comment:
Usually but not for definite! Might be worth a section somewhere detailing this (but perhaps in the whitepaper as this is a more detail-oriented concept).